server/lib/icon.coffee

```diff
@@ -6,6 +6,7 @@ log = require('printit')
 module.exports = icons = {}
 market = require './market'
 
+APPS_FOLDER = '/' + path.join 'usr', 'local', 'cozy', 'apps'
```

Comment:
The right way to do is using path. But what is weird is that '/' at the beginning of the string construction.